GVFS/GVFS.UnitTests/Common/GVFSEnlistmentTests.cs

```diff
@@ -0,0 +1,42 @@
+﻿using GVFS.Common.Git;
+using GVFS.Tests.Should;
+using GVFS.UnitTests.Mock.Common;
+using GVFS.UnitTests.Mock.Git;
+using NUnit.Framework;
+
+namespace GVFS.UnitTests.Common
+{
+    [TestFixture]
+    public class GVFSEnlistmentTests
+    {
+        public const string MountId = "85576f54f9ab4388bcdc19b4f6c17696";

```

Comment:
Why are these public?

Reply:
We are very inconsistent here. We have `public` constants in `MoveRenameFileTests` but `private` constants in `MoveRenameFileTests_2` and `protected` constants in `GitRepoTests`.

It does seem like we use `private` more often. This could be something to add to the contributor style guide (#537) and to consolidate into a cleanup PR during an expert week. (not in scope for this PR, except to change this variable.)